### PR TITLE
Modal pop-up

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
In this project 
- A mobile pop up template was created
- Buttons were linked to it
- all cards can be called from script
- a close button added to it

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -21,3 +23,87 @@ window.onload = () => {
...
+
+    const popup = document.querySelector('.pop-up');
+    popup.style.display = 'block';
+
+    // get title of popup
+    const popupTitle = popup.querySelector('.title');
+    const popupImage = popup.querySelector('.pop-img');
+    const popupTech = popup.querySelector('.btns');
+    const popupDesc = popup.querySelector('.pop-text2');
+
```
Comment (hemantso) [inline script.js]: - [ ] The modal pop-up should be in the center for both mobile view and desktop view as per Figma design. Your modal is aligned to left in desktop view and is taking the whole width in mobile view. Kindly try to implement the modal as per Figma's design.

- [ ] The background colour of buttons is white. Kindly try to change the technology button's colour from grey to white.

- [ ]  On desktop view there is less margin between the projected image and description. Kindly check the below comparison and fix it.

- [ ] The icon on the source button should be aligned properly to the text. Kindly check the below design and fix it.

Your modal on Desktop view
image

Figma design for desktop view
image

Your modal pop up on mobile view
image

Figma design for mobile view
image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -202,7 +201,7 @@ <h1 class="professional-h1">Professional Art<br />Printing Data</h1>
           <li class="listons second">bootstrap</li>
           <li class="listons third">ruby</li>
         </ul>
-        <button class="liston-button bb1">See Project</button>
+        <button class="liston-button bb1 cproject">See Project</button>
```
Comment (DanteAlonsoHT) [inline index.html:204]: - [x] Add more padding for your `li` elements in your popup.
- [x] Align the bottom from the main image with the "see live" and the "see source" buttons.
- [x] Add the right font-family for all your popup.
- [x] Add the right "X" close icon according to the Figma design you selected.
- [x] Add blur background when you are in your popup.

Figma design | Your design

image

File: index.html [hunk 3]
```diff
@@ -189,7 +188,7 @@ <h1 class="professional-h1">Professional Art<br />Printing Data</h1>
...
```
Comment (DanteAlonsoHT) [inline index.html:191]: - [x] Add responsive design for your popup, as you can see in the image below, when you are in the mobile version the popup can look better to show the information properly.

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @tufoinnkuo10, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: script.js [hunk 4]
```diff
@@ -21,3 +23,121 @@ window.onload = () => {
...
+buttons.forEach((btn, i) => {
+  btn.addEventListener('click', () => {
+    const project = projectContent[i];
+
+    const popup = document.querySelector('.pop-up');
+    popup.style.display = 'flex';
+
+    // get title of popup
+    const popupTitle = popup.querySelector('.title');
+    const popupImage = popup.querySelector('.pop-img');
```
Comment (Davidosky007) [inline script.js:108]: * Hi, i can see the popup modal for mobile devices isn't still looking like the original design like the previous code reviewer pointed out kindly adjust the modal on mobile view to look as close as possible as the original design 
* [ORIGINAL]
image
* [YOURS]
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @tufoinnkuo10 @MrBrN197 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
* The mobile view popup modal still needs some adjustment kindly make the necessary adjustment so that yours look as close as possible to the original design 🙏 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**